packages/contentstack-import/src/import/modules-js/locales.js

```diff
@@ -127,6 +127,15 @@ module.exports = class ImportLanguages {
         .catch(function (error) {
           // error while importing languages
           fileHelper.writeFileSync(langFailsPath, self.fails);
+          let message_content_type = '';
+          if (error.request !== undefined && JSON.parse(error.request.data).locale !== undefined) {
+            if (JSON.parse(error.request.data).locale.code) {
+              message_content_type =
+                ' Code of the failed language: ' + JSON.parse(error.request.data).locale.code;
+            } 
+            error.errorMessage = error.errorMessage + message_content_type;
+          }
+          log(self.config, formatError(error.errorMessage), 'error');
```

Comment:
@cs-raj Could you please check to see if we need this? It has already been addressed above.

Reply:
@aman19K  changes are done and removed the redundant code